UPGRADE-1.9.md

```diff
@@ -33,3 +33,10 @@ from `slug` to `code`.
     ```
     bin/console doctrine:migrations:version 'CoreBundle/Migrations/Version20201208105207' --add
     ```
+
+1. The base of `Adjustment` class has changed. If you extend your adjustments already(or have them overridden by default, because of Sylius-Standard usage), you should base your Adjustment class on `AdjustmentFQCN`
```

Comment:
```suggestion
1. The base of `Adjustment` class has changed. If you extend your adjustments already(or have them overridden by default, because of Sylius-Standard usage), you should base your Adjustment class on `Adjustment` FQCN
```
?

Reply:
cc @lchrusciel

Reply:
```suggestion
1. The base of the `Adjustment` class has changed. If you extend your adjustments already(or have them overridden by default, because of Sylius-Standard usage), you should base your Adjustment class on `Sylius\Component\Core\Model\Adjustment` instead of `Sylius\Component\Order\Model\Adjustment`.
```
FQCN(full qualified class name) I used as a short cut, as I was too lazy to copy the whole class name in our conversation :P

Reply:
It is so sad for me (because of last comment) to resolve this conversation, that I won't do it 😂